scapy/main.py

```diff
@@ -10,27 +10,23 @@
 from __future__ import absolute_import
 from __future__ import print_function
 
-
-import atexit
-import code
-import getopt
-import glob
-import gzip
+import sys, os, getopt, re, code
+import gzip, glob
 import importlib
 import logging
-import os
 from random import choice
-import re
-import sys
 import types
 
+try:
+    import IPython # Allow testing
+except:
+    pass
 
-from scapy.config import conf
+# Do not add any imports here that would trigger a warning messsage
```

Comment:
You mean: 'never ever add imports' or 'never add imports before this line' ? I think that you should make this textt more explicit.

Reply:
# Never add global imports, in main.py, that would trigger a warning messsage
# Before the console handlers gets added in interact()